subprojects/docs/src/samples/play/advanced/build.gradle

```diff
@@ -19,6 +19,8 @@ model {
 // START SNIPPET play-dependencies
 dependencies {
     play "commons-lang:commons-lang:2.6"
+    play "com.typesafe.play:play-guice_2.12:2.6.15"
+    play "ch.qos.logback:logback-classic:1.2.3"
```

Comment:
Play 2.6 needs these dependencies to function.

See https://www.playframework.com/documentation/2.4.x/ScalaRouting#Dependency-Injection

Also, since 2.5.0, `Failed to load class "org.slf4j.impl.StaticLoggerBinder"` calls for `logback` dependency.